### PR TITLE
creating a stub of the file for hb ca

### PR DESCRIPTION
**Pull request by wayuyao**
Adding subsections for what we want to know about HB CA

### CODE REVIEW

File: places/hb-ca.md [hunk 1]
```diff
@@ -0,0 +1 @@
+here is some text
```
Comment (helenxmzhou) [inline places/hb-ca.md:1]: Just have some fun !

**Comment by Rosanne6** [comment]
Great job.

File: places/hb-ca.md (hunk 1 above)
Comment (wayuyao) [inline places/hb-ca.md:1]: @helenxmzhou ok, thanks

**Comment by helenxmzhou** [comment]
@wayuyao changes look good

**Pull request merged**